boot-core/src/daemon/state.rs

```diff
@@ -9,10 +9,19 @@ use cosmwasm_std::Addr;
 use ibc_chain_registry::chain::{Apis, ChainData as RegistryChainInfo, FeeToken, FeeTokens, Grpc};
 use serde::{Deserialize, Serialize};
 use serde_json::{from_reader, json, Value};
-use std::{collections::HashMap, env, fs::File, rc::Rc, str::FromStr};
+use std::{collections::HashMap, env, fs::{File, OpenOptions}, rc::Rc, str::FromStr};
 use tonic::transport::{Channel, ClientTlsConfig};
 pub const DEFAULT_DEPLOYMENT: &str = "default";
 
+/*
+    the proper way of using DaemonOptions is using DaemonOptionsBuilder
+    here is an example of how:
+    let options = DaemonOptionsBuilder::default()
```

Comment:
You can use ``` ``` to put code into your doc comments to make them look nice! 
Give this a read when you have some time: https://doc.rust-lang.org/rust-by-example/meta/doc.html

Reply:
*tripple backtick haha